S3_log_bucket/locals.tf

```diff
@@ -2,6 +2,7 @@ locals {
   common_tags = {
     (var.billing_tag_key)  = var.billing_tag_value
     Terraform              = "true"
+    Log_Bucket             = lookup(var.tags, "Log_Bucket", "true")
```

Comment:
Shouldn't this just default to true all the time, when would we want it to be false?